modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view.jsp

```diff
@@ -24,13 +24,8 @@ ClientExtensionAdminDisplayContext clientExtensionAdminDisplayContext = (ClientE
 	actionParameterName="clientExtensionEntryId"
 	creationMenu="<%= clientExtensionAdminDisplayContext.getCreationMenu() %>"
 	dataProviderKey="<%= ClientExtensionAdminFDSNames.CLIENT_EXTENSION_ENTRIES %>"
-	formName="fm"
 	id="<%= ClientExtensionAdminFDSNames.CLIENT_EXTENSION_ENTRIES %>"
 	itemsPerPage="<%= 10 %>"
-	namespace="<%= liferayPortletResponse.getNamespace() %>"
-	pageNumber="<%= 1 %>"
-	portletURL="<%= clientExtensionAdminDisplayContext.getCurrentPortletURL() %>"
 	selectedItemsKey="clientExtensionEntryId"
-	selectionType="multiple"
```

Comment:
I wonder why was this attr active 🤔 
Maybe an unfortunately copy&paste?

Reply:
Yeah, probably :slightly_smiling_face: